common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -98,8 +98,8 @@ def handle_ajax(self, dispatch, data):
         try:
             result = handlers[dispatch](data)
 
-        except NotFoundError as err:
-            log.exception(
+        except NotFoundError:
+            log.info(
                 "Unable to find data when dispatching %s to %s for user %s",
```

Comment:
This is basically the capa equivalent of a 404, right? I'm not sure how I feel about suppressing these to be INFO level. Do we see these often?